aea/manager.py

```diff
@@ -327,13 +332,13 @@ def remove_project(
         """Remove agent project."""
         if public_id not in self._projects:
             raise ValueError(f"Project {public_id} is not present!")
-
         if self._projects[public_id].agents:
             raise ValueError(
                 f"Can not remove projects with aliases exists: {self._projects[public_id].agents}"
             )
 
         project = self._projects.pop(public_id)
+        self._projects_set.remove(public_id.to_latest())
```

Comment:
I don't think this is a good idea. It is not necessarily the latest project which is being added!